documentation/source/user_section/tutorials/registration-to-template/template-registration.rst

```diff
@@ -15,4 +15,16 @@ This tutorial demonstrates how to `register <http://jpeelle.net/mri/image_proces
    template-registration/applying-the-warping-fields
 
 .. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/template-registration/registration-pipeline.png
-   :align: center
\ No newline at end of file
+   :align: center
+
+..
+   comment:: In the SCT Course slides and ``batch_single_subject.sh`` script,
+             we include some additional registration commands to demonstrate
+             other potential uses of ``sct_register_to_template`` (``-param``,
+             ``-ldisc``, compressed cord). I think we are missing these
+             commands from this tutorial. We should consider adding them here.
+             There is a "Customizing Registration" page, however. But, if we
+             follow what was proposed in issue #3095, those tips could be added
+             to ``sct_register_to_template``'s dedicated page, freeing up space
+             in the tutorial for an "other sample commands" page that then
+             links to ``sct_register_to_template``.
```

Comment:
re: 1f84e6dfa37831f73a9a812f35f90a3552e0c483; We might want to rewrite the "Customizing Registration" section and potentially move it to the `sct_register_to_template.rst` page.